tests/InterNations/Tests/Sniffs/Syntax/Fixtures/MethodTypeHints/MissingParameterTypeHint.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+class MissingParameterTypeHint
+{
+    /**
+     * @return Integer[]
```

Comment:
Should be lowercase. Now that I see this, it should be `int[]`. Should we enforce what types to use as well?